src/features/jobs/components/Jobs/JobInfoCard.css

```
@@ -1,25 +1,68 @@
 .cea-job-info-card {
   background-color: rgba(255, 255, 255, 1);
-  padding: 8px;
+  padding: 4px;
+  padding-inline: 8px;
   border-radius: 12px;
   font-size: 14px;
 
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)